### ISSUE TITLE
kube-up.sh doesn't detect Kubernetes installation

### ISSUE THREAD

**Issue by acobster** [deis/workflow#688]
I followed the [Booting Kubernetes Using Vagrant](https://deis.com/docs/workflow/quickstart/provider/vagrant/boot/) instructions and I hit while the box was booting up:

```
$ ./cluster/kube-up.sh
...
==> master: It looks as if you don't have a compiled version of Kubernetes.  If you
==> master: are running from a clone of the git repo, please run 'make quick-release'.
==> master: Note that this requires having Docker installed.  If you are running 
==> master: from a release tarball, something is wrong.  Look at 
==> master: http://kubernetes.io/ for information on how to contact the development team for help.
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

Full output is [here](https://gist.github.com/acobster/dd9d34ba83faa9153188272994e50f14).

I can't verify whether `kubectl` is in the path or anything from within Vagrant because of the SSH issue.

I tried removing the box altogether and rerunning `./cluster/kube-up.sh`, and the issue persists.

Thanks!

**Comment by bacongobbler** [comment]
Can you try running from source instead? We'll have to update this but since it's in maintenance mode things are bound to break. Try the following:

```
git clone https://github.com/kubernetes/kubernetes
cd kubernetes
make quick-release
```

Then follow through with the instructions again.

**Comment by acobster** [comment]
Will do, thanks!

**Comment by acobster** [comment]
Okay, that workaround is up and running for me. Thanks! Let me know if there's any more information I can provide to help fix the underlying issue.

**Comment by bacongobbler** [comment]
Just to make sure, were you copy/pasting the URL in the docs for fetching Kubernetes? I just noticed it was for a k8s v1.3.5 release. Not an issue but that's out of date compared to what master's tested on (v1.5.1) so I'll update the docs.

I manually verified by pulling down the v1.5.1 release on my laptop got me running. I'll update the docs and close this out with that PR, but if someone else reports a similar issue then let's re-open and do some more diving :)
